### ISSUE TITLE
[feature request] multi disk support 

### ISSUE THREAD

**Issue by ghost** [kkrruumm/void-install-script#4]
It whould be really nice if you add multi disk support in this installer

**Comment by kkrruumm** [comment]
@voidbruxo Hey, sorry it's taken me so long to actually get to this -

In the case of multi-disk support, assuming you mean RAID, or if you just mean splitting off something like /home to a different disk, I think the best way for me to go about this is allowing users to set their own mounts with preconfigured disks, so one would set up their disk(s) how they want and then point the installer at them.

I will need to redo a bit of the partitioner to achieve this, but I just want to let you know that I'm looking into it.

If this isn't exactly what you were asking for or if you have any other questions/requests, do let me know.

**Comment by voidbruxo** [comment]
yes that was what I was looking for it , thanks for your attention .
